aca_preview/preview.py

```diff
@@ -336,3 +340,66 @@ def check_bright_guide_for_ers(self, n_bright_req=3, bright_lim=9.0):
         n_bright = np.count_nonzero(self.guides['mag'] < bright_lim)
         if not is_OR and n_bright < n_bright_req:
             self.add_message('critical', f'ER bright stars: only {n_bright} stars brighter than {bright_lim}')
+
+    def check_enough_guide_for_ers(self, n_required=8):
+        """
+        Warn on ERs with fewer than n_required (8) guide stars.
+        """
+        obsid = float(self.obsid)
```

Comment:
Probably makes sense to just do this float cast in `__init__`.  I can't think of a reason we need the str version at this point.